gateway_code/static/iot-lab-cn-m3.cfg

```diff
@@ -1,14 +1 @@
-adapter_khz 1000
-
-# comstick ftdi device
-interface ft2232
-ft2232_layout "usbjtag"
-ft2232_device_desc "ControlNode"
-ft2232_vid_pid 0x0403 0x6010
-
-jtag_nsrst_delay 100
-jtag_ntrst_delay 100
-
-# use combined on interfaces or targets that can't set TRST/SRST separately
-reset_config trst_and_srst
-
+ftdi_device_desc "ControlNode"
```

Comment:
It took me a few seconds for that one. But I re-read the file name and understood :D